README.md

```diff
@@ -127,6 +128,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
 
 <!-- markdownlint-enable -->
 <!-- prettier-ignore-end -->
+
```

Comment:
this currently breaks the welcome page if introduced, so if we're going to do this reformatting we need to fix the welcome page's parsing of the community updates

Reply:
it's not very easy to test

Reply:
I will correct the fetch of the starting page